cmd/testpeers.go

```diff
@@ -4,22 +4,48 @@ package cmd
 
 import (
 	"context"
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
 	"io"
-	"math/rand"
+	"math"
+	"math/big"
+	"os"
+	"os/signal"
+	"slices"
+	"strings"
 	"time"
 
+	k1 "github.com/decred/dcrd/dcrec/secp256k1/v4"
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 	"golang.org/x/exp/maps"
 
 	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/log"
+	"github.com/obolnetwork/charon/app/peerinfo"
+	"github.com/obolnetwork/charon/app/version"
+	"github.com/obolnetwork/charon/app/z"
+	"github.com/obolnetwork/charon/eth2util/enr"
+	"github.com/obolnetwork/charon/p2p"
 )
 
 type testPeersConfig struct {
 	testConfig
-	ENRs      []string
-	P2PRelays []string
+	ENRs             []string
+	P2P              p2p.Config
+	Log              log.Config
+	DataDir          string
+	KeepAlive        time.Duration
+	LoadTestDuration time.Duration
 }
 
+const timeoutInterruptedErr = "timeout/interrupted"
```

Comment:
Best practice is to make it of type Error, in this case we can test against it using `errors.Is` for example.